components/SwipeListView.js

```diff
@@ -256,6 +257,12 @@ SwipeListView.propTypes = {
 	 * past to trigger the row opening.
 	 */
 	swipeToOpenPercent: PropTypes.number,
+	/**
+	 * Describes how much the ending velocity of the gesture affects whether the swipe will result in the item being closed or open.
+	 * A velocity factor of 0 means that the velocity will have no bearing on whether the swipe settles on a closed or open position
+	 * and it'll just take into consideration the swipeToOpenPercent.
+	 */
```

Comment:
Can you add these descriptions to the README as well? I'd also love it if you could add in your note about 5-15 being the ideal number. I was testing this myself and didn't know what to use until I read that

Reply:
will do.